### DISCUSSION TITLE
Purpose of the "Details" Column?

### DISCUSSION THREAD

**Discussion by AndreRatzenberger** [microsoft/DurableFunctionsMonitor#166] [General]
Hi there!

I'm really enjoying the VS Code Extension, especially the ability to deploy it into an online environment—truly awesome!

However, I'm curious about the purpose of the "Details" column. How can I populate it with values? Specifically, I'd like to see the payloads of "EventRaised" entries within this column.

Is there a way to achieve this? Could you please explain the intended use of this column, especially since I actually have never seen anything in it?

<img width="1489" alt="Screenshot 2024-03-14 at 03 35 21" src="https://github.com/microsoft/DurableFunctionsMonitor/assets/44863088/469743ad-1074-4c36-8f1e-fc4086723fda">

Thanks in advance!
Cheers!

**Comment by scale-tone** [comment]
Hi @AndreRatzenberger , the 'Details' column is intended for errors and exceptions of failed steps. 
E.g.:
![image](https://github.com/microsoft/DurableFunctionsMonitor/assets/5447190/53d1b9fa-14bd-409b-8c9e-7e0b192affb2)
